client/verta/verta/_internal_utils/time_utils/_time_utils_py2.py

```diff
@@ -35,6 +35,8 @@ def epoch_millis(dt):
         return int(round((dt - UNIX_EPOCH).total_seconds() * 1000))
     elif type(dt) == int:
         return dt
+    elif dt is None:
+        return dt
```

Comment:
@daniel-verta Do you think this is reasonable to have in this function, or should the caller be expected to handle `None`?

Reply:
I think you've got a reasonable point and we can expect the caller to wrap this in `maybe` if the parameter is optional. So many of our use cases will switch to `maybe(time_utils.epoch_millis, dt)`. We might also rename this to something like `convert_to_epoch_millis` in order to make it clear that it's a function and not a variable, since it could easily look like that in those use cases.

Reply:
That switch isn't pressing in my mind and could be done as part of the ongoing refactoring and clean-up work instead of in this PR depending on your bandwidth.

Reply:
I forgot we had `maybe()`! Amazing.

Reply:
Unlike Scala where `Maybe/Optional` is a standard thing, this would be a new thing for users to have to learn and get used to. The Pythonic way is to have `None` or a value.

Reply:
> a new thing for users to have to learn

Our end users would not be calling `maybe(). This is only about how the client is internally handling optional requirements.